src/Task.js

```diff
@@ -57,7 +57,6 @@ class Task {
 
             taskInfo.parameters.forEach((param) => {
               parameters[param.name] = param;
-              delete parameters[param.name].name;
```

Comment:
This is why I have lobbied from the beginning (5 years maybe?) that task.parameters should not be an object, but an array of parameter objects, each with its own name field.